test/markdown-lint-config.rc

```diff
@@ -0,0 +1,8 @@
+# For help, see
+# https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md 
```

Comment:
<!--whitespace-bot-->
Remove trailing whitespace:
```suggestion
# https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md
```

Reply:
lol linting the markdown lint config